src/platform-includes/distributed-tracing/other-services/javascript.mdx

```diff
@@ -0,0 +1,3 @@
+## Setup Your Other Services
```

Comment:
Should we link to the `tracePropagationTargets` description to tell folks about how they need to configure this option to allow tracing headers being sent to cross-origin services?

Reply:
I have now linked to another page that describes cors and `tracePropagationTargets` so this is done.